pkg/datapath/file_system.go

```diff
@@ -133,9 +133,13 @@ func (fs *fileSystemBR) StartBackup(source AccessPoint, realSource string, paren
 	if !fs.initialized {
 		return errors.New("file system data path is not initialized")
 	}
+	volMode := uploader.PersistentVolumeFilesystem
```

Comment:
would it be interesting to have this in a function and use in both places?
```go
func getPersistentVolumeMode(source AccessPoint) PersistentVolumeMode {
    if source.ByBlock != "" {
	 return uploader.PersistentVolumeBlock
    }
    return uploader.PersistentVolumeFilesystem
}
```

Reply:
I could see adding this as a helper!

Reply:
Updated and added as helper